### ISSUE TITLE
Import Studio, errors on BrickLink references on common parts

### ISSUE THREAD

**Issue by paramecie** [rgriebl/brickstore#308]
I noticed various errors while importing Studio files.
I located one to understand the problem - the 4589b 1x1 cone.

I created in Studio (latest) a file with a single part, a 4589b White cone: it shows 4589b.
I did the same with Leocad (latest), to be sure - it shows also a 4589.dat

But when I import the Studio file in BrickStore, it shows a 59900 (along with an error):
![image](https://user-images.githubusercontent.com/32078594/111412339-88253900-86dc-11eb-99f5-815e62df4ccc.png)

59900 is an alternate reference:
https://www.bricklink.com/v2/catalog/catalogitem.page?P=4589b#T=P

The interesting point is here.

When examining the 2 files in the Studio, it seems one has the only maybe "canonical" reference (59900), and the second one has the BrickLink equivalent reference (the good one, 4589b):

![image](https://user-images.githubusercontent.com/32078594/111412712-39c46a00-86dd-11eb-82bf-5925794baf40.png)

**In short, I think instead of using model.ldr for the import, BrickStore may better use model2.ldr which SEEMS to have more proper BrickLink references.**

Of course, there's a sort of library inside this model2.ldr file which should be skipped, but I guess that's trivial. 

That may solve many/all the errors we have in importing a studio file, and they're not rare - maybe 5% of the items?

This is all deducted, though...

**Comment by rgriebl** [comment]
model2.ldr is really weird. I tried to use it and got totally wrong colors. Turns out it uses BL part ids AND BL color ids: it's a real Frankenstein format, but ideal for import into BrickStore :)

**Comment by paramecie** [comment]
> but ideal for import into BrickStore :)

So, just do it :factory_worker: :yellow_heart:

**Comment by paramecie** [comment]
Another, so common:

![image](https://user-images.githubusercontent.com/32078594/111552688-28ce3400-8783-11eb-8fc5-d903f12ce35c.png)

![image](https://user-images.githubusercontent.com/32078594/111552738-38e61380-8783-11eb-9d99-90af45ce878e.png)

**Comment by paramecie** [comment]
Thanks thanks! I'll try...

**Comment by paramecie** [comment]
Wow - Faaaaaar better! Zero error for now on the file I recently used.
